kwc-challenge-set-listing.html

```diff
@@ -8,27 +9,80 @@
 -->
 
 <dom-module id="kwc-challenge-set-listing">
-  <template>
-    <style>
-      :host {
-        display: block;
-      }
-    </style>
-    <h2>Hello [[prop1]]!</h2>
-  </template>
-
-  <script>
-    Polymer({
-
-      is: 'kwc-challenge-set-listing',
-
-      properties: {
-        prop1: {
-          type: String,
-          value: 'kwc-challenge-set-listing',
-        },
-      },
-
-    });
-  </script>
-</dom-module>
+    <template>
+        <style>
+             :host {
+                display: block;
+            }
+            kano-display-training {
+                margin: auto;
+                max-width: 700px;
+            }
+        </style>
+
+        <template is="dom-repeat" items="[[challengeSets]]" as="challengeSet">
+
+            <kwc-challenge-set
+                label="Challenge Set"
+                challenge-set="[[challengeSet]]"
+                progress="[[_computeCategoryProgress(challengeSet.category, completedChallenges)]]"
+                avatar-url="[[_userAvatarUrl]]"
+            >
```

Comment:
Yo! This little fella here told me he wants to get together with the other dudes on the previous line? We haven't written this anywhere but usually we close the tag after the last property and break the line before closing tags... How do you feel about it?

Reply:
Obviously I prefer to do it like this (that's why I did it) but its not a big deal. Will change it.